contrib/examples/actions/cloudslang-basic.yaml

```diff
@@ -0,0 +1,13 @@
+---
+description: Greet StackStorm from cloudslang ;) !
+enabled: true
+entry_point: ""
+name: cloudslang-basic
+pack: examples
+parameters:
+  flow_path:
```

Comment:
done.
changed to `"/opt/stackstorm/packs/examples/actions/workflows/print_text.sl"` and added the flow to this place.
I think we should re-consider this approach when we will start the next phase of this integration.